apps/wasm/mvp/web/pb/messages.js

```diff
@@ -0,0 +1,332 @@
+/*eslint-disable*/
```

Comment:
This is a generated file right? I think we should include a header warning people that it is generated and should not be edited manually.

We could also consider giving it a different file extension e.g. messages.autogen.js

Reply:
Added the header in the custom codegen script since the codegen tool itself does not provide such an option.

> giving it a different file extension 

That will make the import a little ugly like `import { PeerStatus } from "./pb/messages.autogen";`, I guess that can be implied by the `/pb/` prefix